### ISSUE TITLE
Cut feature does not use vertical bounds for deletion

### ISSUE THREAD

**Issue by stai1** [Sadret/openrct2-scenery-manager#49]
The cut feature deletes the selection without using vertical bounds. This means that if there are active vertical bounds and the selection is cut and pasted, the pasted result does not contain all of the objects that were cut.

It looks like the vertical bounds are applied upon pasting only. Perhaps applying them at copy and deletion time can be considered, too.

**Comment by Sadret** [comment]
You are right, the vertical bounds only apply to pasting. It would make more sense to also apply it to the cut operation.
